frontend/src/scenes/trends/viz/ActionsPie.tsx

```diff
@@ -87,6 +87,7 @@ export function ActionsPie({
                                           date_to,
                                           filters: filtersParam,
                                           breakdown_value,
+                                          seriesId,
```

Comment:
I might not be reading this right, but isn't `seriesId` always the same as `point.index` in the context of pie charts?